ui/lib/core/addon/components/secret-list-header.hbs

```diff
@@ -24,10 +24,7 @@
         <Icon @name={{@model.icon}} @size="24" class="has-text-grey-light" />
         {{@model.id}}
         {{#if this.isKV}}
-          <span class="tag" data-test-kv-version-badge>
-            version
-            {{or @model.version "1"}}
-          </span>
+          <Hds::Badge @text={{concat "version " (or @model.version "1")}} data-test-kv-version-badge />
```

Comment:
This is another one that doesn't need `concat` if you prefer to write it this way:
```suggestion
          <Hds::Badge @text="version {{or @model.version "1"}}"  data-test-kv-version-badge />
```